### DISCUSSION TITLE
IPSec does not connect over VLAN

### DISCUSSION THREAD

**Discussion by i1409** [strongswan/strongswan#1965] [General]
Did somebody has that issue?
I've got a IPSec tunnel for labs purposes and was working ok but then with the vlan implementation on the client's network it does not connect anymore.

When PC is outside of vlan it connects again.

Config on VM

config setup
       charondebug=all

conn %default
    ikelifetime=60m
    keylife=20m
    rekeymargin=3m
    keyingtries=3
    keyexchange=ikev2
    authby=secret
    mobike=no
    fragmentation=no
    tfc=%mtu

conn wan_srv
        auto=start
        ike=aes128-sha256-modp1024!
        esp=aes128-sha256-modp1024!
	
	left=%any4
	leftid="dn=labvm"
	leftsourceip=%config

        right=<peer public address>
        rightid=<peer public address>
        rightsubnet=<remote subnet>

I did change the MTU to 1200 on vif trying to decrease the packet size in order to compense vlan tag space on the packet  but no difference was made.

Thanks

**Comment by tobiasbrunner** [comment]
You probably messed something fundamental up with your network config, so try to fix that. Anyway, you have to clarify what's not working exactly (see #196).
